pkg/reconciler/v1alpha1/serverlessservice/serverlessservice.go

```diff
@@ -0,0 +1,342 @@
+/*
+   Copyright 2019 The Knative Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package serverlessservice
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/google/go-cmp/cmp"
+	"github.com/knative/pkg/controller"
+	"github.com/knative/pkg/logging"
+	netv1alpha1 "github.com/knative/serving/pkg/apis/networking/v1alpha1"
+	informers "github.com/knative/serving/pkg/client/informers/externalversions/networking/v1alpha1"
+	listers "github.com/knative/serving/pkg/client/listers/networking/v1alpha1"
+	rbase "github.com/knative/serving/pkg/reconciler"
+	"github.com/knative/serving/pkg/reconciler/v1alpha1/serverlessservice/resources"
+	"github.com/knative/serving/pkg/reconciler/v1alpha1/serverlessservice/resources/names"
+	"go.uber.org/zap"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1informers "k8s.io/client-go/informers/core/v1"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+)
+
+const (
+	controllerAgentName = "serverlessservice-controller"
+	reconcilerName      = "ServerlessServices"
+	activatorService    = "activator-service"
+	activatorServiceNS  = "knative-serving"
+
+	networkingGroup = "networking.internal.knative.dev"
+	sksLabel        = networkingGroup + "/serverlessservice"
```

Comment:
Why not in `pkg/apis/networking` somewhere?

Reply:
moved.